sys/include/net/wot/serialization/json_keys.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2021 Philipp-Alexander Blum <philipp-blum@jakiku.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#ifndef NET_WOT_SERIALIZATION_JSON_KEYS_H
+#define NET_WOT_SERIALIZATION_JSON_KEYS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+//Todo: Continue reducing extensive usage of duplicated obj keys etc.
+//Todo: Possible to combine multiple const chars. Write function for that.
+// TODO: Definitions might have to be moved to a .c file
+const char _json_key_at_context[] = "@context";
+const char _wot_td_url[] = "https://www.w3.org/2019/wot/td/v1";
+
+const char _obj_key_at_type[] = "@type";
+const char _json_key_titles[] = "titles";
+const char _json_key_descriptions[] = "descriptions";
+const char _json_key_in[] = "in";
+const char _json_key_name[] = "name";
+const char _json_key_qop[] = "qop";
+// const char _json_key_scheme[] = "scheme";
```

Comment:
@Citrullin I had to comment this line out due to a "multiple definitions" error – would it make sense to move the declarations in this file to a separate `.c` file?

Reply:
It probably makes sense to just remove it in json_writer.c. 
Guess I just didn't finished this part.